.github/docs/wiki/_Footer.md

```diff
@@ -1,4 +1,4 @@
-[![Python 3.9+](https://img.shields.io/badge/python-3.9+-blue.svg)](https://www.python.org/downloads/)
+[![Python 11+](https://img.shields.io/badge/python-3.11+-blue.svg)](https://www.python.org/downloads/)
```

Comment:
```suggestion
[![Python 3.11+](https://img.shields.io/badge/python-3.11+-blue.svg)](https://www.python.org/downloads/)
```

Reply:
lol will we ever be alive to see Python 11